src/model/Offset.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * Patch API V1
+ * The core API used to integrate with Patch's service
+ *
+ * Contact: developers@usepatch.com
+ */
+
+import ApiClient from '../ApiClient';
+
+class Offset {
```

Comment:
I think this is useful / good to expose semi-publicly since this is part of the order response.